aspnet/tutorials/first-mvc-app/start-mvc.rst

```diff
@@ -1,16 +1,23 @@
-Getting started with ASP.NET Core MVC
-==================================================
+Getting started with ASP.NET MVC Core
+==============================================
 
 By `Rick Anderson`_
 
-This tutorial will teach you the basics of building an ASP.NET Core MVC app using `Visual Studio`_.
+This tutorial will teach you the basics of building an ASP.NET MVC Core web app using `Visual Studio 2015 <https://www.visualstudio.com/en-us/visual-studio-homepage-vs.aspx>`__. 
+
+
+Install Visual Studio and .NET Core
+----------------------------------------
+
+- Install Visual Studio Community 2015. Select the Community download and the default installation. Skip this step if you have Visual Studio 2015 installed.
+
+  - `Visual Studio 2015 Home page installer  <https://www.visualstudio.com/en-us/visual-studio-homepage-vs.aspx>`__
+
+- Install `.NET Core + Visual Studio tooling <http://go.microsoft.com/fwlink/?LinkID=798306>`__
 
-.. contents:: Sections:
```

Comment:
There are only two sections, **Install** and  **Create a web app** - **install** is short and **Create a web app** is very visible right below it. So it's just clutter.

I don't mind restoring it if that's your pref. - @danroth27  let me know if I should restore it.